src/pages/ProfilePage/ProfilePage.jsx

```diff
@@ -14,12 +13,11 @@ function ProfilePage() {
 	const [friendsPressed, setFriendsPressed] = useState(false);
 	const [userName, setUserName] = useState(""); // State to store the user's name
 	const [friendsNumber, setFriendsNumber] = useState(0); // State to store the number of friends
+	const [Wishlist, setWishlist] = useState([]);
+	const [Favourites, setFavourites] = useState([]);
 
-	useEffect(() => {
-		const currentUser = Parse.User.current();
+	const fetchFriendsNumber = async (currentUser) => {
 		if (currentUser) {
-			setUserName(currentUser.get("name") || "Anonymous"); // dunno if necessary
-
```

Comment:
this disables the functionality so can't be deleted. Please don't delete stuff that others created, as we agreed after I changed something I wasn't supposed to :D with this iteration, if the user has no name set up then there will be no name displayed, not 'anonymous', not 'no name' just simply nothing.

Reply:
I accidentally deleted it! I changed it in the next commit :)